architecture/additional_concepts/storage.adoc

```diff
@@ -351,18 +302,17 @@ For example, Ceph offers *ReadWriteOnce* access mode. You must
 mark the claims as `read-only` if you want to use the volume's
 ROX capability. Errors in the provider show up at runtime as mount errors.
 
-ifdef::openshift-enterprise,openshift-origin[]
```

Comment:
@tmorriso-rh - this looks like an ifdef that was removed by mistake. This is what is causing the build to fail.